server/configuration.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+const defaultValues = {
+  WEB_MONITORING_DB_URL: 'https://api-staging.monitoring.envirodatagov.org'
```

Comment:
Both will work, but anywhere we used to have the `herokuapp.com` URLs, we should probably switch to `envirodata.org`. That way, if we move the underlying service around, everything will keep working.

It’s not in `.env` because this is specifically a default for when someone fails to specify it in their `.env` (since we explicitly note all fields are optional). We have a similar setup in `services/web-monitoring-db.js`, but now that we make requests on the server, we need it here. Speaking of which, I could probably remove it from the client code now. 🤔